pkgs/native_toolchain_c/lib/src/cbuilder/linker_options.dart

```diff
@@ -167,6 +169,16 @@ extension LinkerOptionsExt on LinkerOptions {
     if (gcSections) '/OPT:REF',
   ];
 
+  /// This creates a list of exported symbols - otherwise, some symbols might be
+  /// kept because of `global-dont-strip`.
```

Comment:
As far as I can tell we don't explicitly set `global-dont-strip`. Is that a option on by default?

Reply:
This is not an option - I added some more context

Reply:
I see. Thanks for the added context.